agent/consul/operator_raft_endpoint_test.go

```diff
@@ -113,6 +121,10 @@ func TestOperator_RaftGetConfiguration_ACLDeny(t *testing.T) {
 	if len(future.Configuration().Servers) != 1 {
 		t.Fatalf("bad: %v", future.Configuration().Servers)
 	}
```

Comment:
Forgot to mention, it would be nice to see the tests exercise the "non-leader" output since that includes most of the newly added conditional logic for "Trails Leader By".

Reply:
Apologies, I meant to highlight L50-52. I'll re-open a comment there.